lib/codegen/fromcto/csharp/csharpvisitor.js

```diff
@@ -384,7 +384,7 @@ class CSharpVisitor {
         } else if (!field.isPrimitive()) {
             let fqn = this.getDotNetNamespaceOfType(field.getFullyQualifiedTypeName(), field.getParent(), parameters);
             const modelFile = field.getModelFile();
-            if (modelFile instanceof ModelFile && modelFile.isImportedType(fieldType)) {
+            if (modelFile?.isImportedType?.(fieldType)) {
```

Comment:
`modelFile?.isImportedType(fieldType)`

No need for the second optional check

Reply:
Other way , We can modify the test case above

Reply:
Update the test cases that break. `isImportedType` is meant to be a validation.